docs/references/protocol/transactions/types/ammclawback.md

```diff
@@ -0,0 +1,70 @@
+# AMMClawback
+
+<!--> Add link to github source after feature is merged into devnet. <-->
+[[Source]](https://github.com/XRPLF/rippled/blob/master/src/xrpld/app/tx/detail/AMMClawback.cpp "Source")
+
+_(Added by the [AMM amendment][])_
+
+Claw back tokens from a holder that has deposited your issued tokens into an AMM pool.
```

Comment:
Are holders people...not necessarily...but still seems this should be "Claw back tokens from a holder who...."